docs/farming-&-staking/staking/operators.mdx

```diff
@@ -262,22 +262,27 @@ services:
       - "0.0.0.0:30433:30433/tcp"
     restart: unless-stopped
     command: [
-# Make sure to replace your_domain_id and your_operator_id with your domain ID and operator ID
-# Adjust the keystore path to the location where you generated your operator keypair.
       "--chain", "gemini-3g",
-      "-- ",
-      "--domain-id", "your_domain_id",
-      "--operator-id", "your_operator_id",
-      "--bootnodes", "/ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp",
-      "--bootnodes", "/ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1",
-      "--bootnodes", "/ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7",
+      "--base-path", "/var/subspace",
+      "--port", "30333",
+      "--dsn-listen-on", "/ip4/0.0.0.0/udp/30433/quic-v1",
+      "--dsn-listen-on", "/ip4/0.0.0.0/tcp/30433",
       "--listen-addr", "/ip4/0.0.0.0/tcp/40333",
       "--rpc-cors", "all",
```

Comment:
We only need the `rpc-` parameters if we are connecting a farmer. And this is the pure operator example

Reply:
Left in the farmer example, removed from the pure operator.